modules/client/examples/06-multisig-client/08-get-proposals.ts

```diff
@@ -0,0 +1,63 @@
+/* MARKDOWN
+### Loading the list of proposals (multisig plugin)
+*/
+import {
+  MultisigProposalListItem,
+  MultisigClient,
+  Context,
+  ContextPlugin,
+  IProposalQueryParams,
+  ProposalSortBy,
+  ProposalStatus,
+  SortDirection,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an multisig client
+const client = new MultisigClient(contextPlugin);
+
+const queryParams: IProposalQueryParams = {
+  skip: 0, // optional
+  limit: 10, // optional,
+  direction: SortDirection.ASC, // optional
+  sortBy: ProposalSortBy.POPULARITY, //optional
```

Comment:
Is this actually used, if graphQL doesn't support it?

Reply:
They are going to need it in the app team afaik, for us popularity equals the proposals with more votes, I guess in this case wold be with more approvals?